week2/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+)
+
+func RemoveSpecialCharacter(text string) string {
+
+	// Make a Regex to say we only want letters and numbers
+	reg, err := regexp.Compile("[^a-zA-Z0-9]+")
```

Comment:
`regexp.Compile` has very poor performance, better to create a global variable or inject as input